tests/test_yaml_file_reference.py

```diff
@@ -0,0 +1,58 @@
+#################################################################
+#   Libraries
+#################################################################
+import sys, os
+import pytest
+from pipeline_utils.lib import yaml_parser
+
+#################################################################
+#   Tests
+#################################################################
+def test_file_reference():
+    """
+    """
+    res = [
+            {
+                "accession": "GAPFIXRDPDK5",
+                "aliases": ["cgap-core:reference_genome_hg38"],
+                "description": "hg38 full reference genome plus decoy for CGAP, fasta format",
+                "extra_files": ["fa_fai", "dict"],
+                "file_format": "fa",
+                "institution": "/institutions/hms-dbmi/",
+                "project": "/projects/cgap-core/",
+                "status": "uploading",
+                "uuid": "1936f246-22e1-45dc-bb5c-9cfd55537fe7"
+            },
+            {
+                "aliases": ["cgap-core:reference_genome_hg38"],
+                "description": "hg38 full reference genome plus decoy for CGAP, fasta format",
+                "extra_files": [],
+                "file_format": "fa",
+                "institution": "/institutions/hms-dbmi/",
+                "project": "/projects/cgap-core/",
+                "status": None
+            }
+        ]
+
+    for i, d in enumerate(yaml_parser.load_yaml('tests/repo_correct/portal_objects/file_reference.yaml')):
```

Comment:
I would use `pytest.mark.parametrize` to do this sort of thing, see: https://github.com/dbmi-bgm/cgap-portal/blob/master/src/encoded/tests/test_indexing.py#L345